tools/doc/json.js

```diff
@@ -257,7 +257,7 @@ function processList(section) {
                         JSON.stringify(list));
       }
       current.textRaw = current.textRaw || '';
-      current.textRaw += `${tok.text} `;
+      current.textRaw = `${current.textRaw}${tok.text} `;
```

Comment:
Maybe a slightly better way of doing this would be to make it all one line, like so:

```js
current.textRaw = `${current.textRaw || ''}${tok.text} `;
```